app/routes/goal_routes.py

```diff
@@ -0,0 +1,90 @@
+from flask import Blueprint, request, jsonify, make_response, abort
+from app.models.goal import Goal
+from app.models.task import Task
+from .routes_helpers import validate_model, delete_message
+from app import db
+
+goals_bp = Blueprint("goals", __name__, url_prefix="/goals")
+
+# CREATE GOAL ENDPOINT
+@goals_bp.route("", methods=["POST"])
+def create_goal():
+    request_body = request.get_json()
+
+    if not request_body.get("title"):
+        abort(make_response({"details": "Invalid data"}, 400))
```

Comment:
Good job returning a response if `title` is missing from the request body! Just a note about messaging - think of the response details like what might the client need to know about the unsuccessful request? "Invalid data" is accurate but not precise enough to be helpful. Imagine the client might need to show a message about missing params somehow, but "Invalid data" doesn't tell the client what was missing!